templates/default/kibana.yml.erb

```diff
@@ -88,12 +87,18 @@ elasticsearch.password: <%= @config['elasticsearch.password'] %>
 #elasticsearch.password: "pass"
 <% end -%>
 
-# Paths to the PEM-format SSL certificate and SSL key files, respectively. These
-# files enable SSL for outgoing requests from the Kibana server to the browser.
+
+# Enables SSL and paths to the PEM-format SSL certificate and SSL key files, respectively.
+# These settings enable SSL for outgoing requests from the Kibana server to the browser.
+<% if @config.key?('server.ssl.enabled') %>
+server.ssl.enabled: <%= @config['server.ssl.enabled'] %>
```

Comment:
these are new options of Kibana configuration? what about backwards compatibility? what will change for current users?

Reply:
Do you consider thats ok to keep both of them ?

Reply:
Hi @anuriq. Do you want to remove the new settings and to merge the pull request?

Reply:
Yes, please remove this commit and submit it as a separate PR so we can make sure to bump versions/defaults at the same time.